checkout.py

```diff
@@ -3,11 +3,15 @@
 Created on Fri Aug 13 11:35:57 2021
 
 @author: vjham
+
 """
+from datetime import datetime
 from validation import is_clerk_input_valid
 
 items = []
 GROCERY_STORE_NAME = "Shop N Bag"
+DATE_AND_TIME = datetime.now()
```

Comment:
So the actual date is not a constant - (I know that can be a little confusing, because the value never changes after it is set. However, we aren't setting this variable (DATE_AND_TIME) to a "constant" value. We are setting it to whatever the current date is.), so we probably don't want to use all caps for the name here